Assets/Scripts/Characters/PlayerCombat.cs

```diff
@@ -15,9 +15,12 @@ public class PlayerCombat : MonoBehaviour
 
         private void Awake()
         {
-            _playerInput = GetComponentInParent<PlayerInput>();
             _playerInput.actions["Player/Fire"].performed += HandleFire;
+        }
 
+        private void OnEnable()
```

Comment:
I was getting an issue where the input was picking up a click after I started the game in the editor. I moved it down here and it went away although I'm not sure why. We should probably figure out some best practices on enable/start/awake